### ISSUE TITLE
Hunter damage disproportionately affected in Naxxramas

### ISSUE THREAD

**Issue by KrasterK** [Progressive335/Progressive335#666]
I have noticed that every time there is a damage nerf in place hunter damage does down more significantly than others, it happened in BWL when the 35% damage reduction was in place, i dropped to the bottom of the dps meters, and once the nerf was lifted i was on the top 3 again. Same thing is happening now in Naxx, im at the bottom of the dps meters but in other raids with the same people im at the top. I dont know what can be causing this, or why hunters in particular. I always use the same gear and rotation

![WoWScrnShot_032720_225711](https://user-images.githubusercontent.com/57421066/77830737-e56e5600-712a-11ea-815d-8420d3b5dd1d.jpg)
![WoWScrnShot_032720_230605](https://user-images.githubusercontent.com/57421066/77830738-e69f8300-712a-11ea-902d-f33a343db826.jpg)
![WoWScrnShot_032720_232032](https://user-images.githubusercontent.com/57421066/77830739-e7381980-712a-11ea-9942-1ef0faed862d.jpg)
![WoWScrnShot_032720_233010](https://user-images.githubusercontent.com/57421066/77830740-e7381980-712a-11ea-908a-f2a53f94e3c8.jpg)
![WoWScrnShot_032720_234230](https://user-images.githubusercontent.com/57421066/77830741-e7d0b000-712a-11ea-8cd5-21c33ac384b1.jpg)
![WoWScrnShot_032720_234239](https://user-images.githubusercontent.com/57421066/77830742-e7d0b000-712a-11ea-9431-c80eaa83973a.jpg)

These are images of our last BWL run, i will take SS of next Naxx to compare
Thank you for your time.

**Comment by KrasterK** [comment]
After doing some test on target dummies it seems that the damage reductions seems to be caused by armor and crit suppresion in bosses. Against small target dummies level 60 my normal hits are 1k+ damage, and my crits 1,6-2k. On the big target dummy of skull level my normal hits were 500-600 damage and my crits 800-900 damage, which is very similar to what happens in Naxxramas, i crit way less often than what i feel i should and i lose enormous amounts of dps. 
I found information about crit suppression on bosses +3 level over the character is 1,6% per level of difference so 4,8% of the global crit chance of the character.
I also found a table with armor and resistances of vanilla bosses i'll share it in case it helps

[Vanilla Bosses' Defensive Stats.pdf](https://github.com/Progressive335/Progressive335/files/4404328/Vanilla.Bosses.Defensive.Stats.pdf)
Classes that ignore armor seem to do way better than the more physical damage ones, Casters' spell ignore armor completely, Frost DKs do 60% frost damage, Rogues have bleeds, nature damage from poison and other attacks and have debuffs that ignore armor, if they are raid wide maybe the physical damage from ranged is not being affected by the debuff. Also when damage is based on crits once a 50% nerf is applied crits damage is also nerfed 50% which makes it not have a real difference between normal shots and crits since they're so similar with the damage reduction. Hunter damage as MM is only 18-20% which can ignore certain bosses armor depending on the Nature Resistance of that particular boss.

**Comment by KrasterK** [comment]
It seems to be an issue of melee getting no nerf on white damage, but ranged physical still have the white damaged nerfed

**Comment by Gubbyfishy** [comment]
> Against small target dummies level 60 my normal hits are 1k+ damage, and my crits 1,6-2k. On the big target dummy of skull level my normal hits were 500-600 damage and my crits 800-900 damage

there's a good chance the training dummy might be level 83. It's often an oversight on most servers. The disparity between level 60 and presumably 63 is too wide to be true.

Regardless, you scale worse for the same reason you scale significantly better than others when damage mods occur, i.e Nature vulnerbility on Chromaggus with Chimera Shot. You suffer double dampening due to reduced damage on both Serpent Sting and Chimera Shot -Serpent Sting.

TL;DR try survival or maybe even BM since they get affected less. Also, the above mentioned happens to Shadow priests as well.

**Comment by KrasterK** [comment]
@Gubbyfishy thank you so much for your reply, im afraid you might be right, i'm gonna try with survival first since i tried BM and i was still at the bottom.

**Comment by Progressive335** [comment]
> > Against small target dummies level 60 my normal hits are 1k+ damage, and my crits 1,6-2k. On the big target dummy of skull level my normal hits were 500-600 damage and my crits 800-900 damage
> 
> there's a good chance the training dummy might be level 83. It's often an oversight on most servers. The disparity between level 60 and presumably 63 is too wide to be true.
> 
> Regardless, you scale worse for the same reason you scale significantly better than others when damage mods occur, i.e Nature vulnerbility on Chromaggus with Chimera Shot. You suffer double dampening due to reduced damage on both Serpent Sting and Chimera Shot -Serpent Sting.
> 
> TL;DR try survival or maybe even BM since they get affected less. Also, the above mentioned happens to Shadow priests as well.

no, training dummies are scaled to lvl 60 (some to lvl 63)
